docs/src/explanations/moduleprefix.md

```diff
@@ -0,0 +1,122 @@
+# Module Prefixing
```

Comment:
can you add this doc to the explanations TOC (unless that is auto-generated now)? Note that is in 'order in which you should read these'

Reply:
I put it at the end, since it's a rather niche feature.